docs/developers/guard-accounts.md

```diff
@@ -0,0 +1,134 @@
+---
+id: guard-accounts
+title: Guard accounts
+---
+
+[comment]: # (mx-abstract)
+
+On this page, you will find comprehensive information on all aspects of guarded (co-signed) transactions.
+
+[comment]: # (mx-context-auto)
+
+# Introduction
+
+Phishing attacks that scam people have become a constant problem on blockchains in general, especially when newbies to the crypto world are granted easy access to this new economy. In light of these new challenges, it is important to continue educating people on specific ways to protect themselves while also considering new methods to add protection against such attacks. That's how we came to a method that we called Guardians, with the scope of having a guardian for each account.
+
+[comment]: # (mx-context-auto)
+
+# For consumers
+
+## Web-Wallet
+
+## xPortal
+
+There is no solution released for the **xPortal Wallet**. As soon as there is one, it will be documented and referenced here.
+
+## Ledger
+
+There is no solution released for the **Ledger**. As soon as there is one, it will be documented and referenced here.
+
+# For developers
+
+If you are a developer and you understand what a transaction is, what fields it contains and how a signing of a transaction happens you may be interested by the technical aspects of the implementation.  can be achieved by adding another field in the transaction structure for a second signature, and defining the option to flag the transactions that are supposed to be executed/verified as multisig transactions. The option field in the transaction is currently used to differentiate between transactions where the signature should be verified over the marshaled transaction and transactions where the signature should be verified over the hash of the transaction, but additional options could be defined.
+
+The full activation of the extra protection requires two actions:
+
+**1. Register a Guardian**
+
+**2. Guard the account**
+
+After guarding the account, _almost_ every action the user takes should be co-signed by the guardian.
+
+:::note
+**Why not mandate that every transaction is co-signed by a guardian when an account is guarded?**
+
+This is done in order to not have the user locked out of his account (e.g user guards his account but the guardian also loses access to his account). For this reason, the user should not only be able to **_set up a new guardian_**, but also **_change the guardian_** if he wishes so (e.g when the guardian is compromised).
+:::
+
+## Registering & Changing the guardian
+
+
+
+### Register a Guardian
```

Comment:
Set a Guardian?
registers makes me think of creating an account that will act as a guardian